Tests/XiCliCoreTests/XiCliCoreTests.swift

```diff
@@ -90,4 +98,15 @@ class XiCliCoreTests: XCTestCase {
         expectation.fulfill()
     }
 
+    func fileInCurrentDir(_ filename: String) -> String {
+        var url = URL(string: FileManager.default.currentDirectoryPath)!
```

Comment:
There is a convenience initializer that might be helpful here, [`Url.init(fileURLWithPath: String, relativeTo: URL?)`](https://developer.apple.com/documentation/foundation/url/3126803-ini)